LMS/Landing/Repositories/LandingRepository.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace LMS\Landing\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use LMS\Courses\Models\Course;
+use LMS\User\Models\User;
+
+class LandingRepository
+{
+    public function retrieveData(): array
+    {
+        return [
+            'achievements' => [
+                'students' => $this->getStudents(),
+                'contentHours' => $this->getCourseTotalTime(),
+                'watchedLessons' => $this->getTotalWatchedLessons()
+            ],
+            'courses' => [
+                'free' => $this->getFreeCourseInformation(),
+                'paid' => $this->getPaidCourseInformation()
+            ]
+        ];
+    }
+
+    private function getFreeCourseInformation(): array
+    {
+        $model = new Course();
+        $courseTime = 0;
+        foreach ($model->where('paid', false)->get() as $value) {
+            $courseTime += $value->duration;
+        }
+
+        return [
+            'count' => $model->where('paid', false)->count(),
+            'time' => gmdate('H:i:s', $courseTime),
+            'support' => '❌'
```

Comment:
Acredito que o mais correto seria passar um boolean e lá na blade realizar uma condicional para renderizar o emoticon, apenas sugestão de fato
```suggestion
            'support' => 0
```